Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -17,30 +20,125 @@ public class SettingsMenu : PlatformDependentComponent
+                // set the titles of the scrollViews to the scopes
+                TextMeshProUGUI groupTitle = scrollView.transform.Find("Group").gameObject.MustGetComponent<TextMeshProUGUI>();
+                groupTitle.text = $"{group.Key}";
+                // display the scrollview for each scope, with its bindings and keys
+                foreach (var binding in group)
+                {
+                    // instantiates the keyBindingContent
+                    keyBindingContent = PrefabInstantiator.InstantiatePrefab(KeyBindingContent, Canvas.transform, false).transform.gameObject;
+                    keyBindingContent.transform.SetParent(scrollView.transform.Find("Scroll View/Viewport/Content"));
+
+                    // set the text to the bindingName
+                    TextMeshProUGUI bindingText = keyBindingContent.transform.Find("Binding").gameObject.MustGetComponent<TextMeshProUGUI>();
+                    string bindingName = binding.Value.Substring(0, binding.Value.IndexOf("[")).ToString();
+                    bindingText.text = bindingName;
+
+                    // set the label of the key button
+                    TextMeshProUGUI key = keyBindingContent.transform.Find("Key/Text (TMP)").gameObject.MustGetComponent<TextMeshProUGUI>();
+                    key.text = KeyBindings.GetKeyNameForButton(bindingName);
+                    // add the label to the dictionary
+                    buttonToLabel[bindingName] = key;
+                    // add the actionlistener, to be able to change the key of a binding
+                    keyBindingContent.transform.Find("Key").gameObject.MustGetComponent<Button>().onClick.AddListener( () => { StartRebindFor(bindingName); });
+                }
+            }
         }
 
         /// <summary>
-        /// Toggles the settings panel with the ESC button.
+        /// Toggles the settings panel with the ESC button and handels
+        /// the case, that the user wants to change the key of a keyBinding.
         /// </summary>
         protected override void UpdateDesktop()
         {
+            // when the buttonToRebind is not null, then the user clicked a button to start the rebind.
+            if(buttonToRebind != null)
+            {
+                SEEInput.KeyboardShortcutsEnabled = false;
+                // the next button, that gets pressed, will be the new keyBind.
+                if (Input.anyKeyDown)
+                {
+                    // get the key that was pressed.
+                    foreach(KeyCode key in Enum.GetValues(typeof(KeyCode)))
+                    {
+                        //rebind the key
+                        if (Input.GetKeyDown(key))
+                        {
+                            // check if the key is already bound to another binding, if not, then update the key
+                            if(KeyBindings.SetButtonForKey(buttonToRebind, key))
```

Comment:
There must always be exactly one blank between a control keyword and the corresponding opening bracket `(`.